cmd/image-converter/image-converter.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"os/exec"
+
+	"k8s.io/klog/v2"
+)
+
+func main() {
+	var srcVolPath, dstVolPath, srcFormat, dstFormat string
+
+	flag.StringVar(&srcVolPath, "src-path", "", "Source volume path")
+	flag.StringVar(&dstVolPath, "dst-path", "", "Target volume path")
+	flag.StringVar(&srcFormat, "src-format", "", "Format of the source volume")
+	flag.StringVar(&dstFormat, "dst-format", "", "Format of the target volume")
+
+	flag.Parse()
+
+	klog.Info("srcVolPath: ", srcVolPath, " dstVolPath: ", dstVolPath, " sourceFormat: ", srcFormat, " targetFormat: ", dstFormat)
+	err := convert(srcVolPath, dstVolPath, srcFormat, dstFormat)
+	if err != nil {
+		klog.Fatal(err)
+	}
+}
```

Comment:
once/if we integrate progress reporting here then I have no objection for triggering virt-v2v this way